java/src/com/google/template/soy/xliffmsgplugin/XliffResType.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.template.soy.xliffmsgplugin;
+
+enum XliffResType {
+  SELECT("soy-select"),
+  PLURAL("soy-plural");
```

Comment:
http://docs.oasis-open.org/xliff/v1.2/os/xliff-core.html#restype says

> In addition, user-defined values can be used with this attribute. A user-defined value must start with an "x-" prefix.

Should these be `x-soy-...`?